### PR TITLE
nrjmx improvements

### PR DESCRIPTION
**Pull request by cristianciutea**
This PR will contain the changes for improving the nrjmx comunication protocol

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,4 +8,5 @@ TAG				?= v0.0.0
 
 include $(INCLUDE_BUILD_DIR)/build.mk
 include $(INCLUDE_BUILD_DIR)/ci.mk
-include $(INCLUDE_BUILD_DIR)/release.mk
\ No newline at end of file
+include $(INCLUDE_BUILD_DIR)/release.mk
+include $(INCLUDE_BUILD_DIR)/release.mk
```
Comment (rubenruizdegauna) [inline Makefile:12]: duplicated line ☝️

File: build/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 FROM maven:3.6-jdk-8
 
 ARG GH_VERSION='1.1.0'
+ARG GO_VERSION='1.16.1'
```
Comment (rubenruizdegauna) [inline build/Dockerfile:4]: not a big issue but latest `1.16` is `1.16.10` ☝️

File: src/go/go.mod [hunk 3]
```diff
@@ -0,0 +1,11 @@
+module github.com/newrelic/nrjmx
+
+go 1.14
```
Comment (rubenruizdegauna) [inline src/go/go.mod:3]: should we use 1.16 here ☝️  ?

**Review by rubenruizdegauna — APPROVED** [review]
LGTM

**Review by brushknight — APPROVED** [review]
LGTM

**Pull request merged**